src/runtime/schema/schema.ts

```diff
@@ -129,8 +130,16 @@ export function create(appState: AppState): SchemaInternal {
         const nexusSchemaConfig = mapSettingsAndPluginsToNexusSchemaConfig(plugins, settings.data)
         nexusSchemaConfig.types.push(...statefulNexusSchema.state.types)
         nexusSchemaConfig.plugins!.push(...appState.schemaComponent.plugins)
-        const { schema, missingTypes } = NexusSchema.core.makeSchemaInternal(nexusSchemaConfig)
-        return { schema, missingTypes }
+        try {
+          const { schema, missingTypes } = NexusSchema.core.makeSchemaInternal(nexusSchemaConfig)
+          /**
+           * Validate the GraphQL schema
+           */
+          GraphQL.validate(schema, GraphQL.parse(GraphQL.introspectionQuery))
```

Comment:
I'm frankly not sure about this line that I've just added. What I've realized is that we have some cases , such as a type without any field, that aren't caught as a bug. However, when running the server, the playground sends an introspection query, which ends up throwing an unexpected error a couple of seconds later. This line here, tries to validate the schema against an introspection query to validate this class of error. WDYT?